### ISSUE TITLE
Shortcut widget to jump to another sitemap & tab

### ISSUE THREAD

**Issue by anordvall** [igorgladkov/rotini#48]
To allow for one-click access, it would be beneficial to have a Shortcut widget that takes a sitemap name (optional I guess) and tab name as input. Clicking on the widget should take you straight to that sitemap & tab.

**Comment by igorgladkov** [comment]
Can you give few example use cases?

**Comment by MrMontesa** [comment]
I would use this as well. Here's my use-case.
My tablet is wall mounted (Nexus 7 2012) with a neat surrounding frame to only see the display.
Thus I'm not able to do the left-to-right swipe in order to bring up the main menue and select another sitemap. I do have cellar, ground floor & first floor split into sitemaps.
I have to remove the frame, do the menu swipe, choose different sitemap and mount the fram again. Would be very convendient to have a widget that allows me do directly jump to the first-floor sitemap.

**Comment by igorgladkov** [comment]
@MrMontesa I assume that the app is running in full-screen mode in your case, right?

**Comment by MrMontesa** [comment]
@igorgladkov Yes!

**Comment by anordvall** [comment]
Either way it's currently three clicks (and possibly some swiping) to access a specific tab in a specific sitemap. I'm a big fan of the one-click idea :)

**Comment by anordvall** [comment]
My use case is to quickly be able to jump between the main tablet's sitemap and all (or some of) the different tabs in the "rooms" sitemap. Even jumping between tabs in the same sitemap without having to swipe through all tabs would be beneficial.

**Comment by igorgladkov** [comment]
@anordvall Can you give a bit more details about your setup that requires switching between sitemap often?

**Comment by igorgladkov** [comment]
> Even jumping between tabs in the same sitemap without having to swipe through all tabs would be beneficial.

I don't see the difference between tapping on a tab or tapping on a widget within the same sitemap, what are the benefits you fell it brings? I would actually say, that widget is bound to single tab, while tabs are available at all time (e.g. less clicks), so isn't it even less convenient?

**Comment by igorgladkov** [comment]
> My use case is to quickly be able to jump between the main tablet's sitemap and all (or some of) the different tabs in the "rooms" sitemap.

Okay, to confirm, you have one sitemap which is more of generic dashboard with bunch of stuff, and then another sitemap which is room-centric (one tab per room/area), sounds about right?

**Comment by anordvall** [comment]
Yes that's spot on. I could of course duplicate all the room tabs into my generic sitemap... will give that a thought.

One difference between widgets and tabs is that the widgets are always on the same location (in my case as I don't need vertical scrolling) but the tabs needs to be scrolled to access all of them.

**Comment by MrMontesa** [comment]
@anordvall you can also directly click on the tabs to jump to them. No nees to scroll right until you hit the last tab. Simply use the top-tabs as navigation menue.

I do have a generic 1st tab followed by room specific tabs. One sitemap per floor. I also try to avoid up/down scrolling within tabs.

**Comment by anordvall** [comment]
@MrMontesa, yes I know... but still... tabs jumping around can be a bit confusing and it takes some time to find out where a specific room tab is.

Either way, I've now duplicated my configs so I now have my generic dashboard as first tab "Main" and the specific rooms as separate tabs in the same sitemap. For each tablet I install I will need to duplicate the config again as the tablets should have different generic dashboards.

I'm fine with sticking to this approach for now and can live with having to navigate the tabs :)

So, my own immediate need for this feature has decreased. Perhaps leave the issue open for a while to see what others say about it.
